frontend/src/app/modules/invite-user-modal/principal/principal-search.component.ts

```diff
@@ -44,6 +44,7 @@ export class PrincipalSearchComponent extends UntilDestroyedMixin implements OnI
     switchMap(this.loadPrincipalData.bind(this)),
     share(),
   );
+  private emailRegExp:RegExp = /^\S+@\S+\.\S+$/;
```

Comment:
does this match the one in the backend?

Reply:
It matches (almost) the one used in the backend, not for the validation but for the endpoint that does the autocompletion in the "Members" view.